src/bilder/components/docker/steps.py

```diff
@@ -0,0 +1,78 @@
+import json
+from io import StringIO
+
+from pyinfra import host
+from pyinfra.api.deploy import deploy
+from pyinfra.api.exceptions import DeployError
+from pyinfra.api.util import make_hash
+from pyinfra.facts.deb import DebPackages
+from pyinfra.facts.server import Command, LinuxDistribution
+from pyinfra.operations import apt, files
+
+
+def _apt_install():
+
+    apt.packages(
+        name="Ensure Docker CE prerequisites are present",
+        present=True,
+        packages=["curl", "gnupg-agent", "software-properties-common", "lsb-release"],
```

Comment:
You can probably remove `lsb-release` since we're not relying on it any more.

Reply:
:+1: